app/(main)/settings/team/team-members-list.tsx

```diff
@@ -30,12 +25,12 @@ export function TeamMembersList({
 			<div className="divide-y divide-zinc-800">
 				{members.map((member) => (
 					<TeamMemberListItem
-						key={member.userId}
+						key={`${member.userId}-${member.displayName}-${member.role}-${currentUserRole}`}
```

Comment:
If the same user belongs to multiple teams and the key is only userId, the data is not updated after switching teams.
Fixed key configuration to handle cases where referenced data differs by team affiliation.

ref: https://react.dev/learn/you-might-not-need-an-effect#resetting-all-state-when-a-prop-changes

Reply:
@satococoa Thanks 👍  b3cc5e50aece fixed